examples/group/README.md

```diff
@@ -0,0 +1,182 @@
+
+1. **Pipeline input repository**: `labresults` - Group by patient: the group will be done by PATID - Group by hospital: the group will be done by CLIA.
+2. **Pipeline**: Executes a set of command lines creating a new directory named after each capture group and copying the files that match the given group. (See our 2 pipelines:[`lab_group_by_hospital.json`](./lab_group_by_hospital.json) and [`lab_group_by_patient.json`](./lab_group_by_patient.json)).
+3. **Pipeline output repository**: `group_by_hospital`or `group_by_patient` depending on which use case you run - Each output repo will contain a list or sub-directories named after each capture group and populated with a copy of their matching files.
+In the diagram below, we have mapped out the data of our example and the expected results in each case.
+![group_example1](./img/group_example1.png)
+
+***Step 1*** - Prepare your data: Let's first create our mock dataset and create/populate our repository.
+The setup target `setup-lab` of the `Makefile` in `pachyderm/examples/group` will create a directory (labresults) with our example data.
+In the `examples/group` directory, run:
+```shell
+$ make setup-lab
+```
+***Step 2*** - Create/populate Pachyderm's repository and create your pipelines
+In the `examples/joins` directory, run:
+```shell
+$ make deploy-lab
+```
+or run:
+```shell
+$ pachctl create repo labresults
+$ pachctl put file -r labresults@master:/ -f labresults
+$ pachctl create pipeline -f lab_group_by_hospital.json 
+$ pachctl create pipeline -f lab_group_by_patient.json
+```
+Have a quick look at your repositories: 
+```shell
+$ pachctl list file labresults@master
+```
+You should see the following files:
+
+![labresults repo](./img/list_file_labresults_master.png)
+
+The commit in your entry repository has triggered the execution of your pipeline (i.e., a job). Have a quick check at your pipeline's status:
+```shell
+$ pachctl list pipeline
+```
+Once it has run successfully, you should see something like this:
+
+![pipelines](./img/lab_list_pipeline.png)
+
+***Step 3*** - Let's have a look at our final product: Check the output repository of your pipeline.
+```shell
+$ pachctl list file group_by_patient@master
+$ pachctl list file group_by_hospital@master
+```
+You should see your expected sub repositories. 
+
+Check one testresult for patient 1:
+```shell
+$ pachctl get file group_by_patient@master:/1/T1606707613-LIPID-PATID1-CLIA24D9871328.txt
+```
+
+## 4. Example 2 - Group pipeline on several repositories 
+***Goal***
+For each store, we are going to calculate the net amount of all transactions (net_amount = order_total - return_total) and save it to a text file named after the store identifier.
+
+1. **Pipeline input repositories**: `stores` ,`returns`, `purchases` - Group by STOREID on all 3 repositories. Each match (i.e., all transactions (purchases and returns) having occured at a given store along with the store information itself) will generate a datum.
```

Comment:
See above comments on formatting.  I think it's less important here, but it should be consistent. The sentence works better.

I would use `STOREID` everywhere, as well.

Fix misspelling on "occurred"